### ISSUE TITLE
(Voicing) Fine tuning regions for electron accumulation and discharge amounts

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#399]
In a couple of interviews, visual learners kept rubbing beyond the "max amount of electrons on body", so in March 16th design meeting we decided to add a second response after hitting "max amount of electrons on body". That response is:

- John fully charged.

However, @Matthew-Moore240 and I also noticed that the range for the region "several electrons on body" seems too wide, especially for non-visual learners.

So in addition to adding "John fully charged", we would like to also adjust the ranges for the regions. I have updated the tables defining the regions, in the design document in section [Accumulation and Discharge of Electrons](https://docs.google.com/document/d/1nHzqDbRvv-By6gYK59Al345AY7D2z87Ng2im8nf4z2w/edit#heading=h.45132ed7vjce). I am not sure why we didn't notice this before, but the ranges between "several electrons" and "a huge amount" were uneven. I evened them out which should help.

Qualitative Accumulation Amount | New Ranges | New Total Number for Region
-- | -- | --
{{A couple of}} electrons on body. | 1 - 4 | 4
{{A few}} electrons on body. | 1 - 5 | 5
{{Several}} electrons on body. | 10 - 30 | 21
{{A bunch of}} electrons on body. | 31 - 53 | 23
{{A large amount of}} electrons on body. | 54 - 76 | 23
{{A huge amount of}} electrons on body. | 77 - 99 | 23
{{Max amount of}} electrons on body. | = 100 | 100
John fully charged. |   | X number of non-accumulating rubs when already “at max”
John fully charged. |   | X number of non-accumulating rubs when already “at max”

**Note** that the ranges for "a couple", "a few" and "max" are unchanged.

In the design doc I also separated out the table for discharge amounts. These amounts had not been updated in the design doc, but I am assuming they should follow the accumulation amounts.

Updated to use same regions as for accumulation | New Ranges | Adjusted Range for Discharge Amounts
-- | -- | --
{{A couple of}} electrons discharged | 1 - 4 | = or > 4
{{A few}} electrons discharged | 5 - 9 | = or > 5 & < 10
{{Several}} electrons discharged | 10 - 30 | = or > 10 & < 31
{{A bunch of}} electrons discharged | 31 - 53 | = or > 31 & < 54
{{A large amount of}} electrons discharged | 54 - 76 | = or > 54 & < 77
{{A huge amount of}} electrons discharged | 77 - 99 | = or > 77 & < 100
{{Max amount of}} electrons discharged | 100 | = 100

@jessegreenberg, is this all clear? Here's a summary:
1. Adjust the range for electrons accumulation and discharge amounts, so the upper regions are more even.
2. Verify that discharge amounts are using the same scale as accumulation amounts.
3. Add 1 new response two times if rubbing continues past "max amount of electrons".

Let me know if you have any questions.

Note: edited table, and removed hint to move arm. I think this reflects the decision from our March 30th meeting.

**Comment by terracoda** [comment]
@jessegreenberg, I can't find the decision that was made at our March 30 design meeting about how many times to repeat the "John fully charged" context response. I think we decided on two repeats.

The goal is to make sure the learner knows they have reached the max amount of electrons on body, and that they need to try to do something else now, like maybe move the hand, play the hint button or reset the sim. Further rubbing is no longer productive.

In Friction's Interactive Description design, there is logic that occasionally repeats, "Jiggling super fast, super hot". There is also a reset hint that repeats, but that it not relevant for John Travoltage as we are not going to provide a dynamic hint.

From using Friction, I can't seem to figure out if we hard coded a pattern for the repetition, but the repetition happens a couple of times and then falls silent. That's the kind of experience I am aiming for when a learner is rubbing John's foot.

They should hear the max electrons response, and then if they keep rubbing, they'll hear "John fully charged" a couple of times, and then only hear rubbing after that with no voicing. Kind of like this:

I am rubbing the foot and I reach the max amount (100 electrons):
 1. I hear: "Max amount of electrons on body."
 - I keep rubbing for a bit, but do not accumulate more charges and do not get a discharge. 
 2. I hear: "John fully charged."
- I keep rubbing for a bit, and still do not accumulate more charges and do not get a discharge. 
3. Again, I hear: "John fully charged."

If I keep rubbing after that, I won't get any more responses.

I am not that concerned about how many times  "John fully charged." repeats or how often. The main thing is that if unproductive rubbing happens for a bit, I should hear, "John fully charged." a couple of times to gently let me know I should try something else.

The voicing design for Friction will need the same kind of logic, so this is something we can re-use.

**Comment by terracoda** [comment]
@jessegreenberg, let me know if you have any more questions.

**Comment by jessegreenberg** [comment]
This is done and should be ready for review, @terracoda can you please verify this is working as you would expect?
https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.1/phet/john-travoltage_en_phet.html?supportsVoicing

**Comment by terracoda** [comment]
Yes, verified.
All good and I get "John fully charged." 2 times when I try to rub again.
